Runtime/Scripts/ExampleWebSocket.cs

```diff
@@ -54,7 +55,18 @@ async Task StartWebSocket(string url)
             await wsClient.Connect(uri);
         }
 
-
+        private void HandleFindCloudlet(EdgeEventsStatus edgeEventstatus, FindCloudletEvent fcEvent)
+        {
+            print("NewFindCloudlet triggered status is " + edgeEventstatus.status + ", Trigger" + fcEvent.trigger);
+            if(fcEvent.newCloudlet != null)
+            {
+                print("New Cloudlet FQDN: " + fcEvent.newCloudlet.Fqdn);
+            }
+            if(edgeEventstatus.status == Status.error)
+            {
+                print("Error received: " + edgeEventstatus.error_msg);
+            }
```

Comment:
nits: space between "if" and "(". Also check error first

Reply:
Ok, will proceed without checking the error as per our conversation

Reply:
Will track this later here https://mobiledgex.atlassian.net/browse/EDGECLOUD-5229